### ISSUE TITLE
How to make Project Rome on Xamarin (Android) forget about previous auth code?

### ISSUE THREAD

**Issue by MahdiGhiasi** [Microsoft/project-rome#49]
I'm adding a "Log out" button into my Xamarin (Android) app which uses Project Rome.

How can I make `Platform` forget about the current authenticated user so it asks for authentication (via `FetchAuthCode` event) again?

**Comment by MahdiGhiasi** [comment]
Is there any timeline for v0.11 SDK being released for Xamarin for Android too? (Currently it's available on Android, but Xamarin for Android is still at v0.8)

**Comment by cmknox** [comment]
No, we do not have a specific timeframe at this point.  However, it's definitely something we'll be working on soon, so stay tuned!

**Comment by MahdiGhiasi** [comment]
It's been almost two months now, and Project Rome SDK for Xamarin for Android is still at v0.8. Any news?

**Comment by cmknox** [comment]
Hey thanks for checking back!  We are actively investigating work in this space, so yes progress is being made!  Nothing to announce quite yet, however.  Thanks for checking...

**Comment by mediaexplorer74** [comment]
> Is there any specific timeline for v0.11 SDK release on Xamarin for Android? (Currently it's available on Java Android, but Xamarin for Android is still at v0.8)

Hi. I noticed that Xamarin folder (branch) at Project Rome was deleted... Do you have some starter app? Please share any simple X app that simply sends Hello from sender to receiver :)
